x-team/xp-ui
# Create JobPageProTipCard

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3368

## Description

Create `JobPageProTipCard` (I pick this name because it seems something for "JobPages" only and to keep related components next to each other in the navigation... but I don't have strong opinion about whether it should be just `ProTipCard`).

ALso, I added some missing components at the `index.js`. As far as I got the index.js should export all available components, right? Please let me know if that's not correct then I'll revert changes.

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3368-create-prop-tip-card` | [XP Registration](https://github.com/x-team/xp-registration/pull/109)

## Steps to Test or Reproduce

1. Check the component at http://localhost:9001/?path=/story/ui-components-jobspageprotipcard--basic-usage
1. Check JobsPageLayout compositions with the new component included http://localhost:9001/?path=/story/screens-and-layouts-jobspagelayout--basic-usage

## Impacted Areas in Application

New component added `JobPageProTipCard`

## Screenshots

<img width="773" alt="Screen Shot 2019-11-14 at 11 46 32" src="https://user-images.githubusercontent.com/131859/68867832-3c701300-06d5-11ea-941c-a85c79464b0e.png">

![2019-11-14 12 46 29](https://user-images.githubusercontent.com/131859/68872520-ebfcb380-06dc-11ea-93e2-8eb40e4b2204.gif)
